foundation-extensions/adobe-experience-platform-assurance/README.md

```diff
@@ -250,7 +249,8 @@ To start using the extension library, you must first register the extension with
 
         let extensions = [Assurance.self, ...]
         MobileCore.registerExtensions(extensions, {
-		        MobileCore.configureWith(appId: "yourAppId")          
+            // set app id from the Data Collection UI    
+		        MobileCore.configureWith(appId: "yourAppId")  
```

Comment:
formatting - let's fix the alignment for all these occurrences

Reply:
This does look alright in code editor (typora , Atom), I have seen sometimes its the git diff that gets confused with tabs and spaces.. Probably should looks alright when its published (will edit back if not)

Although I did align them properly in other places where it needed it